src/components/internal/button.ts

```diff
@@ -0,0 +1,27 @@
+import {html} from 'lit';
+import {withTwind} from '../twind/withTwind';
+import {LwcElement} from '../lwc-element';
+import {color} from '../css/colors';
+import {gradientText} from '../css/gradientText';
+
+export class Button extends withTwind()(LwcElement) {
```

Comment:
Cool, I didn't know we can directly override existing HTML components. I can't find any docs for this, did you find any? but seems to work.